controller/ctrldbops.py

```diff
@@ -12,7 +12,7 @@ def find_hosts():
     if hosts:
         return hosts
     else:
-        return False
+        return []
```

Comment:
I'm curious as to why...?  I haven't coded anything using python yet... I've coded in PHP in the past, so am just skimming over the code... this jumped out to me as odd... is this done as a preference sort of thing?  To save a couple characters in the code?  What are the benefits of using the shorter syntax?  I prefer the shorter... just curious...

Reply:
So it returns an array always.